Description: Fix include path for the speechd headers for Ubuntu
Author: Matthias Klose
Last-Updated: 2018-09-14

--- a/src/mumble.pri
+++ b/src/mumble.pri
@@ -126,6 +126,8 @@
 		PKG_CONFIG = pkg-config --static
 	}
 
+	INCLUDEPATH *= "/usr/include/speech-dispatcher"
+
 	QMAKE_CFLAGS *= "-I../mumble_proto" "-isystem ../mumble_proto"
 	QMAKE_CXXFLAGS *= "-I../mumble_proto" "-isystem ../mumble_proto"
 }
